src/ui/src/wds/WdsIcons/LucideIcon.vue

```diff
@@ -0,0 +1,48 @@
+<!-- eslint-disable vue/no-v-html -->
+
+<template>
+	<span :style="{ display: 'contents' }" v-html="rawHtml" />
+</template>
+
+<script setup lang="ts">
+import { nextTick, onMounted, ref, useAttrs, watch } from "vue";
+import { createIcons, icons } from "lucide";
+
+defineOptions({
+	inheritAttrs: false,
+});
+
+const props = defineProps({
+	name: { type: String, required: true },
+});
+
+const attrs = useAttrs();
+
+const rawHtml = ref("");
+
+function renderIcon(name: string) {
+	rawHtml.value = `<i data-lucide="${name}"></i>`;
```

Comment:
Do you really need `v-html` ?
```suggestion
	<span :style="{ display: 'contents' }">
	  <i :data-lucide="name"></i>
	</span>
</template>

<script setup lang="ts">
import { nextTick, onMounted, ref, useAttrs, watch } from "vue";
import { createIcons, icons } from "lucide";

defineOptions({
	inheritAttrs: false,
});

const props = defineProps({
	name: { type: String, required: true },
});

const attrs = useAttrs();

const rawHtml = ref("");

function renderIcon(name: string) {
	rawHtml.value = ``;
```

Reply:
Yes, in this case we do need `v-html`

The reason is that some components pass icon names dynamically (e.g. `chevron-down` vs `chevron-up`). And Lucide `createIcons` function replaces the <i> element with actual SVG **only once**.

When Vue re-renders, it doesn't get updated, since Lucide doesn't re-process already replaced elements (no reactive logic).

Using v-html lets us re-inject a fresh <i> node on every change and trigger createIcons() cleanly on nextTick.